ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -60,7 +62,7 @@ public void shouldRunCorrectTest() throws UnsupportedEncodingException {
   }
 
   @Test
-  public void shouldUseAndCloseTestExecutor() {
+  public void shouldUseAndCloseTestExecutor() throws IOException, RestClientException {
```

Comment:
nit: `throws Exception` rather than trying to keep track of individual exceptions in unit tests. Courtesy of @big-andy-coates ;)